src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,11 @@ const Cashier = (() => {
         });
     };
 
+    const showP2PNote = (currency) => {
+        const is_usd_account = currency === 'USD';
```

Comment:
let's name it something more general like `is_currency_allowed` in case we add more supported currencies later